xmtp_mls/src/groups/serial.rs

```diff
@@ -0,0 +1,73 @@
+use openmls::prelude::MlsGroup as OpenMlsGroup;
```

Comment:
This file looks good, will try it later to adapt the changes into my PR and see what would happen!